yt_dlp/extractor/radiko.py

```diff
@@ -183,8 +184,9 @@ class RadikoIE(RadikoBaseIE):
     }]
 
     def _real_extract(self, url):
-        station, video_id = self._match_valid_url(url).groups()
-        vid_int = unified_timestamp(video_id, False)
+        station, timestring = self._match_valid_url(url).groups()
+        video_id = join_nonempty(station, timestring)
```

Comment:
`join_nonempty` isn't really necessary here since neither value could be empty, but it's fine